### DISCUSSION TITLE
[off] Frete correios

### DISCUSSION THREAD

**Discussion by Juuhz** [frontendbr/forum#1430] [Discussão]
Fala devs, tudo joia?

O tópico que estou levantando, irá sair um pouco da curva do front-end, mas acredito que vocês possam me ajudar por conta da vasta experiência de vocês nos mais diversos projetos desenvolvidos. 

Estou trabalhando em um projeto de e-commerce, mas meu cliente está com uma dúvida de como conseguir um frete mais barato, exemplo:

O produto dele custa: R$ 20,00 ( São adesivos, super leves ).

Mas o frete entre RIOxSP ( por exemplo ), já fica em torno de 17 a 23 reais pelos correios, oque é relativamente caro por conta do baixo valor do produto. 

----

Como meu cliente poderia oferecer um frete mais baratos para o usuário final ?

Abraços <3

**Comment by vicainelli** [comment]
Cara, não sei se o serviço de [Carta Comercial](http://www.correios.com.br/precos-e-prazos/servicos-nacionais/carta) dos Correios atende seu cliente, mas os valores são bem baixos pra cartas com gramatura inferior a 300gr (que é o que um envelope com adesivos deve pesar)

**Comment by Juuhz** [comment]
Irado demais @vicainelli, valeu pela dica, acredito que se enquadra certinho. 

Só uma dúvida: Os preços informados no site do correios, são para o brasil inteiro ? Sabe me dizer?

**Comment by vicainelli** [comment]
@Juuhz pelas simulações que fiz, a tabela se matém, independente do Estado/Cidade

**Comment by Juuhz** [comment]
Show, valeu pela ajuda @vicainelli
